artifactory/commands/transferfiles/utils.go

```diff
@@ -237,7 +238,7 @@ func sendSyncChunksRequest(curTokensBatch UploadChunksStatusBody, chunksLifeCycl
 	curTokensBatch.AwaitingStatusChunks = chunksLifeCycleManager.GetInProgressTokensSlice()
 	curTokensBatch.ChunksToDelete = chunksLifeCycleManager.deletedChunksSet.ToSlice()
 	chunksStatus, err := srcUpService.syncChunks(curTokensBatch)
-	if err != nil {
+	if err != nil && !errors.Is(err, context.Canceled) {
```

Comment:
Let's add a comment explaining why we skip the errors of type `context.Canceled`.